config/environments/development.rb

```diff
@@ -76,7 +76,7 @@
 
   config.hosts << ENV["APP_DOMAIN"] unless ENV["APP_DOMAIN"].nil?
   if (gitpod_workspace_url = ENV["GITPOD_WORKSPACE_URL"])
-    config.hosts << URI.parse(gitpod_workspace_url).host
+    config.hosts << /.*#{URI.parse(gitpod_workspace_url).host}/
```

Comment:
Since you mentioned `port-<workspace-url>`, do we want to be more specific here, i.e.`\d*` instead of `.*`?